### ISSUE TITLE
mubbus.cpp bugfix

### ISSUE THREAD

**Issue by GoogleCodeExporter** [luizcantoni/mudbus#5]
```
I noticed an error in your library. Line 137 should be MessageLength = 12;
I was getting an error when writing to coils as the response was incorrect.


```

Original issue reported on code.google.com by `dunners...@gmail.com` on 6 Apr 2015 at 4:34

**Comment by kasperv79** [comment]
are these corrections verified?

**Comment by vhu474** [comment]
YES. Confirmed and verified to fix the issue on my Weintek HMI screen Modbus. Please merge this into main branch.
ByteArray[4 to 5] corresponds to message length + 1(Unit Identifier). On main, its 2, which is clearly wrong

MessageLength is 8 on main, which i dont know how they got. The header itself is 8 bytes, so its not including the message.



Source: https://www.prosoft-technology.com/kb/assets/intro_modbustcp.pdf

**Comment by vhu474** [comment]
Also, this is the official repo that Controllino uses for their HMIs. I hope this issue gets fixed.
